xscen/extract.py

```diff
@@ -628,6 +624,10 @@ def search_data_catalogs(
             f"Removing {len(ex.df)} assets based on exclusion dict : {exclusions}."
         )
 
+    if match_hist_and_fut:
```

Comment:
Wait. How to you select an experiment with this new order ?
If I want "rcp45", I would I put it in `other_search_criteria`, does this mean I now have to say : `experiment=['historical', 'rcp45']` ?

Reply:
Right... I didn't use `experiment` in my test, but it would indeed be an issue. Do you have an idea on a possible compromise? This order is a lot faster, but I'm not against reverting this change if we need to.

Reply:
Alright, let's forget it for now, then.